rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixDc.java

```diff
@@ -65,6 +66,8 @@ public class InformixDc extends AbstractDbDc {
 
     private final MetricsCollector metricCollector;
 
+    public boolean sqlTraceEnabled;
```

Comment:
Do you need it to be public? I don't see you are using it anywhere outside this class.

```suggestion
    private boolean isSqlTraceEnabled;
```